test/math/flightMath.spec.js

```diff
@@ -26,11 +27,22 @@ ava('.calcTurnRadius() returns a turn radius based on speed and bank angle', t =
     t.true(result === expectedResult);
 });
 
-ava('.calculateHeadingFromTwoPositions()', t => {
+ava('.bearingToPoint() returns the bearing from one point to another', t => {
     const positionStart = [-99.76521626690608, -148.0266530993096];
     const positionEnd = [-87.64380662924125, -129.57471627889475];
     const expectedResult = 0.5812231343277809;
-    const result = calculateHeadingFromTwoPositions(positionEnd, positionStart);
+    const result = bearingToPoint(positionStart, positionEnd);
 
     t.true(result === expectedResult);
 });
+
+ava('.fixRadialDist() returns a point defined by a direction and distance away from another point', t => {
+    const positionStart = [36.455167, -121.879667];
+    const radial = 6.02139;
+    const distance_nm = 8.5;
+    const expectedResult = [36.59190833043389, -121.92530325256963];
+    const result = fixRadialDist(positionStart, radial, distance_nm);
+
+    t.true(result[0] === expectedResult[0]);
+    t.true(result[1] === expectedResult[1]);
```

Comment:
👍 👍 👍 
Look at you, adding tests.  Hell yeah!!

_fun fact_: When I'm comparing arrays or objects within tests, sometimes I use `_isEqual()` to help me out.  You did good with this test, I like the verbosity.  I just want to make you aware of it because there may come a time when you need to compare larger arrays/objects.   As an example, using your test above: 
```javascript
t.true(_isEqual(result, expectedResult));
```

You don't need to change anything, I like this test the way it is.  Just something to keep in mind if you run into that.  Bringing in lodash functions to help in tests is totally cool.